scripts/create-example-tests.js

```diff
@@ -625,8 +625,20 @@ ${rows}
   );
 
   const referenceQueryable = Queryable.from('reference', referencesParsed);
-  const examplePathOriginal = referenceQueryable.where({ refId: 'reference' }).value;
+  const examplePathOriginal = referenceQueryable.where({ refId: 'reference' })
+    ? referenceQueryable.where({ refId: 'reference' }).value
+    : '';
+  if (!examplePathOriginal) {
+    log.error(`ERROR: Valid 'reference' value not found in "${directory}/data/references.csv".`);
+    process.exit(1);
```

Comment:
This `process.exit` can be removed. `log.error` already does this.

Reply:
Done